java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -305,14 +309,20 @@ int maxSteps() {
   }
 
   private void execute(String signature, SquidClassLoader classLoader) {
-    programState = ProgramState.EMPTY_STATE;
-    steps = 0;
-    BytecodeCFGBuilder.BytecodeCFG bytecodeCFG = BytecodeCFGBuilder.buildCFG(signature, classLoader);
-    if (bytecodeCFG == null) {
+    BytecodeCFGMethodVisitor cfgVisitor = new BytecodeCFGMethodVisitor();
+    MethodLookup lookup = MethodLookup.lookup(signature, classLoader, cfgVisitor);
+    if (lookup == null) {
+      LOG.debug("Method lookup failed for %s", signature);
```

Comment:
Message is confusing : method was not found and it is harmless to the analysis, I would not use the word fail in this kind of message.